src/typo3/customize.md

```diff
@@ -198,18 +200,18 @@ plugin.tx_aimeos.settings.i18n.cz.0 {
 
 ## Special characters
 
-Sometimes, the source string, that should be translated, contains special characters like new lines (\n) or backslashes (\). This is no problem, if you place them in a .po file of your project specific extension (same location as the original .po file) and transform its content using the Unix Gettext command, e.g.:
+Sometimes, the source string, that should be translated, contains special characters like new lines (\n) or backslashes (\). This is no problem if you place them in a ".po" file of your project specific extension (same location as the original ".po" file) and transform its content using the Unix Gettext command, e.g.:
 
 ```bash
 msgfmt -c -o de de.po
 ```
 
-Using them in TypoScript is more difficult, because TypoScript doesn't allow new lines in values. Therefore, you have to use "\n" (a backslash and the character "n") as replacement:
+Using them in TypoScript is more difficult because TypoScript doesn't allow new lines in values. Therefore, you have to use "\n" (a backslash and the character "n") as replacement:
 
 ```typoscript
 plugin.tx_aimeos.settings.i18n.de.0 {
   domain = client
-  string = The payment was canceled.\nDo you wish to retry your order?
+  string = The payment was cancelled.\nWould you like to place your order again?
```

Comment:
This is a string from the code which shouldn't be changed here

Reply:
OK. (But these strings will be fixed, too, and then we have to come back again...)